lib/URLButton.js

```diff
@@ -0,0 +1,130 @@
+//reimplimentation of add-on sdk button/toggle, found at
```

Comment:
Oops, didn't mean to include this file in the commit. It seems to me that WebExtension only has some of the functionality that we need, such as pageAction's ability to put button in nav bar, but not other critical functionality, like attaching a pop-up panel to that button. Is there an easier way to combine features of sdk and webExtension?